### DISCUSSION TITLE
Setting a specific flag in an itemgroup

### DISCUSSION THREAD

**Discussion by NetSysFire** [CleverRaven/Cataclysm-DDA#58451] [Q&A] [answered]
I am currently working on renovating the awful orc "village" in Magiclysm, which is just a giant ugly square with 3 doors right now. I want to place some fitting orc foodstuffs into the village for flavor (no pun intended), including smoked human meat. There are some `conditional_names`  in e.g `meat_smoked`, but how do I make sure the items that get spawned are specifically made from human flesh? This is for lore and environmental storytelling purposes.

I have seen `variants` in `ITEM_SPAWN.md` but this does not seem to be it.

**Comment by NetSysFire** [comment, marked as answer]
Solution:

```json
  {
    "id": "demo",
    "type": "item_group",
    "subtype": "collection",
    "entries": [
      { "item": "meat_smoked", "count-max": 3, "custom-flags": "CANNIBALISM", "prob": 70 }
    ]
  }
```

**Reply by NetSysFire** [reply]
However, there is a limitation I found. This does not work for e.g `jerky` because it gets spawned in a container, namely a plastic bag and the content does not get the flags applied.

1. This should be fixed.
2. There should also be an option to not spawn an item with the container.

**Reply by GuardianDll** [reply]
yeah, it doesnt work, sadly, even when it shows no errors
see https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/mapgen_palettes/aircraft_carrier_palette.json#L227
